session-v7.go

```diff
@@ -32,14 +32,14 @@ import (
 	"github.com/minio/minio/pkg/quick"
 )
 
-func migrateSessionV5ToV6() {
+func migrateSessionV6ToV7() {
```

Comment:
but I know that in general, sequential migration is a good idea to avoid bugs and hidden problems